docs/usage/self-hosted-configuration.md

```diff
@@ -99,15 +112,16 @@ All text inside the start and end `/` will be treated as a regular expression.
 
 ```json
 {
-  "autodiscoverFilter": "/project/.*/"
+  "autodiscoverFilter": ["/project/.*/"]
 }
 ```
 
-You can negate the regex by putting a `!` in front:
+You can negate the regex by putting a `!` in front.
+If using negations, all repositories except those who match the regex are added to the result:
```

Comment:
```suggestion
Only use a single negation and don't mix with other filters because all filters are combined with `or`.
If using negations, all repositories except those who match the regex are added to the result:
```
i think we need to clarify behavior here.